sencha-workspace/SlateDemonstrationsStudent/app/controller/Dashboard.js

```diff
@@ -22,7 +22,10 @@ Ext.define('SlateDemonstrationsStudent.controller.Dashboard', {
         studentSelector: 'slate-demonstrations-student-dashboard slate-appheader slate-cbl-studentselector',
         contentAreaSelector: 'slate-demonstrations-student-dashboard slate-appheader slate-cbl-contentareaselector',
         competenciesSummary: 'slate-demonstrations-student-competenciessummary',
-        cardsCt: 'slate-demonstrations-student-cardsct'
+        recentProgress: 'slate-demonstrations-student-recentprogress',
+        levelsLegend: 'slate-cbl-levelslegend',
+        cardsCt: 'slate-demonstrations-student-cardsct',
+        nonEnrollmentMessage: 'slate-demonstrations-student-dashboard container[itemId="non_enrollment_message"]'
```

Comment:
FYI, this is the preferred way to utilize an itemId in a selector:

```suggestion
        nonEnrollmentMessage: 'slate-demonstrations-student-dashboard #non_enrollment_message'
```

But see other feedback, none of these selectors are needed